bazel/init/stage_3.bzl

```diff
@@ -14,5 +15,18 @@ def stage_3():
     * A transitive load statement that references a repository that doesn't
       exist until stage 2 completes
     """
-    python_deps()
+
+    pip_parse(
+        name = "python_dependencies",
```

Comment:
This name has to be unique between enkit, internal, and all other repos - how about prefixing it? `enkit_pip_deps` or something

Reply:
Yeah, that makes sense. I will do that.